src/common/components/hosts/utils.ts

```diff
@@ -113,12 +114,13 @@ export const canHostnameBeChanged = (hostStatus: Host['status']) =>
     'pending-for-input',
   ].includes(hostStatus);
 
-export const getHostRole = (host: Host, schedulableMasters?: boolean): string => {
+export const getHostRole = (host: Host, t?: TFunction, schedulableMasters?: boolean): string => {
```

Comment:
why optional ?

Reply:
Because when we use in OCM we don't have translations at the moment